### ISSUE TITLE
Better phrasing for "Users table" in setup script

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#16489]
In the setup script, there is a configuration directive that sets `$cfg['Servers'][$i]['users']`, which is used as part of the Configuration Storage feature for managing user groups. The phrasing "Users table" is pretty ambiguous. Is there something better we can use here? Something like "User groups users table" seems like only a slight improvement — is "Users table for user group feature" too wordy? 

First brought up at https://hosted.weblate.org/translate/phpmyadmin/master/en/?&offset=682

**Comment by mauriciofauth** [comment]
![Screenshot_2020-11-20 phpMyAdmin setup](https://user-images.githubusercontent.com/120970/99817475-bd08f100-2b2b-11eb-9adf-599e1e8feef4.png)

**Comment by lem9** [comment]
In the setup script we see:

Users table
User groups table

How about this:

User groups : users table
User groups : groups table
